metrics/web/test/dashboard/presentation/widgets/build_results_metric_graph_test.dart

```diff
@@ -80,75 +78,32 @@ void main() {
     );
 
     testWidgets(
-      "applies a text style from the build result bar graph theme data",
+      "displays a date range widget with the date range view model if it is not null",
```

Comment:
Would it be great to divide this test into 2 separate tests:
- displays the date range widget if the date range view model is not null
- applies the date range view model to the date range widget

What do you think?